public/stylesheets/style.css

```diff
@@ -175,3 +177,8 @@ h6 {
   border: 0 !important;
   border-radius: 0 !important;
 }
+
+.btn:focus, .btn-large:focus, .btn-floating:focus {
+  background-color: crimson;
+  color: white;
+}
```

Comment:
Line contains following spacing inconsistencies:
- No newline at EOF.

*Origin: SpaceConsistencyBear, Section: `cli`.*

The issue can be fixed by applying the following patch:

```diff
--- a/public/stylesheets/style.css
+++ b/public/stylesheets/style.css
@@ -181,4 +181,4 @@
 .btn:focus, .btn-large:focus, .btn-floating:focus {
   background-color: crimson;
   color: white;
-}+}
```